### ISSUE TITLE
Gaia standards failing color cuts

### ISSUE THREAD

**Issue by sbailey** [desihub/desispec#1151]
sv1rosette tile 80718 has 186 standards on petal 5 according to https://data.desi.lbl.gov/desi/target/fiberassign/tiles/trunk/080/fiberassign-080718.png .  The spectro pipeline apparently considered 50 of these and discarded all of them due to the `|delta color|<0.1` cut, which smells suspicious.  Please investigate why the pipeline didn't find any good standards for this petal.
```
desi_compute_fluxcalibration --delta-color-cut 0.1 \
--infile /global/cfs/cdirs/desi/spectro/redux/cascades/exposures/20210130/00073720/frame-b5-00073720.fits \
--sky /global/cfs/cdirs/desi/spectro/redux/cascades/exposures/20210130/00073720/sky-b5-00073720.fits \
--fiberflat /global/cfs/cdirs/desi/spectro/desi_spectro_calib/trunk/spec/sm9/fiberflatnight-b5-20201214.fits \
--models /global/cfs/cdirs/desi/spectro/redux/cascades/exposures/20210130/00073720/stdstars-5-00073720.fits \
--outfile $SCRATCH/fluxcalib-b5-00073720.fits 
```
Other petals just barely made it, e.g. sp0 discards 49/50:
```
desi_compute_fluxcalibration  --delta-color-cut 0.1 \
--infile /global/cfs/cdirs/desi/spectro/redux/cascades/exposures/20210130/00073720/frame-b0-00073720.fits \
--sky /global/cfs/cdirs/desi/spectro/redux/cascades/exposures/20210130/00073720/sky-b0-00073720.fits \
--fiberflat /global/cfs/cdirs/desi/spectro/desi_spectro_calib/trunk/spec/sm4/fiberflatnight-b0-20201214.fits \
--models /global/cfs/cdirs/desi/spectro/redux/cascades/exposures/20210130/00073720/stdstars-0-00073720.fits \
--outfile $SCRATCH/fluxcalib-b0-00073720.fits
```
@segasai please investigate this as a followup to your implementation of supporting Gaia standards.

**Comment by segasai** [comment]
Thanks, I'll take a look in next couple of days. A quick check shows that extinction in the field is EBV=0.5, So I'd guess issues must be connected to that.

**Comment by segasai** [comment]
I've created a PR #1159 that fixed the issue. (after some investigation, I think I can trace the problem to the EBV scaling. The dust_transmission correction of the model spectra needs to use the corrected EBV. If it's not, then the wrong template sets are used (due to the color selection of templates) and that leads to them being discarded at the -fluxcalibration stage.

**Comment by segasai** [comment]
Just a remark -- the change will affect the non-Gaia-only standard fields with significant extinction (I don't know if there were many).
And obviously with the change applied desi_compute_fluxcalibration succeeds on the problematic tile  with 26 stars selected.

**Comment by julienguy** [comment]
solved with PR #1269
